pyartcd/pyartcd/util.py

```diff
@@ -267,3 +268,28 @@ async def is_build_permitted(version: str, data_path: str = constants.OCP_BUILD_
 
     # Fallback to default
     return True
+
+
+def get_release_controller_arch(release_stream_name):
+    arch = 'amd64'
+    streamNameComponents = release_stream_name.split('-')
+    for goArch in goArches:
+        if goArch in streamNameComponents:
+            arch = goArch
+    return arch
+
+
+def get_release_controller_url(release_stream_name):
+    arch = get_release_controller_arch(release_stream_name)
+    return f"https://{arch}.ocp.releases.ci.openshift.org"
+
+
+def log_dir_tree(path_to_dir):
+    for child in os.listdir(path_to_dir):
+        child_path = os.path.join(path_to_dir, child)
+        logger.info(child_path)
+
+
+def log_file_content(path_to_file):
```

Comment:
Same as above, would be nice to log an explanatory message such as `Printing file contents for <path>` beforehand

Reply:
add info logs